types/swagger-ui-dist/index.d.ts

```diff
@@ -18,5 +13,33 @@ export as namespace SwaggerUIDist;
 export function getAbsoluteFSPath(): string;
 export function absolutePath(): string;
 
-export const SwaggerUIBundle: any;
+interface SwaggerUIBundle {
+  (a: {
+    deepLinking: boolean;
+    dom_id: string;
+    presets: any[];
+    plugins: any;
+    urls: Array<[string, string]>;
+    layout: string;
+  }): any;
```

Comment:
Some params are missing. e.g. spec, url


Parameter name | Docker variable | Description
-- | -- | --
configUrl | CONFIG_URL | String. URL to fetch external configuration document from.
dom_id | DOM_ID | String, REQUIRED if domNode is not provided. The ID of a DOM element inside which SwaggerUI will put its user interface.
domNode | Unavailable | Element, REQUIRED if dom_id is not provided. The HTML DOM element inside which SwaggerUI will put its user interface. Overrides dom_id.
spec | SPEC | Object={}. A JavaScript object describing the OpenAPI definition. When used, the url parameter will not be parsed. This is useful for testing manually-generated definitions without hosting them.
url | URL | String. The URL pointing to API definition (normally swagger.json or swagger.yaml). Will be ignored if urls or spec is used.
urls | URLS | Array. An array of API definition objects ([{url: "<url1>", name: "<name1>"},{url: "<url2>", name: "<name2>"}]) used by Topbar plugin. When used and Topbar plugin is enabled, the url parameter will not be parsed. Names and URLs must be unique among all items in this array, since they're used as identifiers.
urls.primaryName | URLS_PRIMARY_NAME | String. When using urls, you can use this subparameter. If the value matches the name of a spec provided in urls, that spec will be displayed when Swagger UI loads, instead of defaulting to the first spec in urls.

Reply:
https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md